packages/completer/src/tokens.ts

```diff
@@ -212,6 +212,11 @@ export interface IInlineCompletionItem extends IInlineCompletionItemLSP {
    * with `token` which has to be set for incomplete completions.
    */
   isIncomplete?: boolean;
+
+  /**
+   * This field is marked when an error occurs during a stream or fetch request.
+   */
+  isError?: boolean;
```

Comment:
To consider:
- should it be a boolean or maybe an object so that we can add "message" and "type" in the future?
- should it be a field at all, or should we rely on throwing errors in stream/fetch methods and catching them in reconciliator to populate ghost text instead?

Reply:
In LSP spec error is two levels higher, in the response message which contains the `result` (and optionally `error` which is a `ResponseError` object). https://microsoft.github.io/language-server-protocol/specifications/lsp/3.18/specification/#responseMessage

But the error should be defined per item. Maybe it is fine to have `error?: CompletionError` field here.